packages/commonwealth/server/controllers/server_proposal_methods/get_completed_proposal_votes.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  ServerProposalsController,
+  SupportedProposalNetworks,
+} from '../server_proposals_controller';
+import { providers } from 'ethers';
+import { ChainNetwork } from 'common-common/src/types';
+import { formatAaveProposalVote, getAaveProposalVotes } from './aave/votes';
+import { ServerError } from 'near-api-js/lib/utils/rpc_errors';
+import { IAaveVoteResponse } from 'adapters/chain/aave/types';
+
+export type GetProposalVotesOptions = {
+  chainId: string;
+  proposalId: number;
+};
+
+export type GetProposalVotesResult = any[];
+
+export async function __getProposalVotes(
```

Comment:
it looks like the filename and method name are not consistent.

Can you either rename the method to `__getCompletedProposalVotes` or rename the file to `get_proposal_votes.ts`?

Reply:
Fixed filenames and function names in the controller.